### PR TITLE
simulator: remove `balance-region-scheduler` util all stores put the regions to pd

### LINKED ISSUE #5251: simulator: remove `balance-region-scheduler` util all store put the regions to pd.
## Development Task
In past, every node has different initial tick, so the pd will balance some region to the later store at first. related code:

https://github.com/tikv/pd/blob/3695d81648005cd0f7eb566cdfcda3248c9c3565/tools/pd-simulator/simulator/node.go#L80-L90

store status:
image

operators:
image

### PR DESCRIPTION
**Pull request by bufferflies**
Signed-off-by: bufferflies <1045931706@qq.com>

### What problem does this PR solve?

Issue Number: Close #5251,#5290

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test (add detailed scripts or steps below)

Code changes

Side effects

Related changes

### Release note

```release-note
 None.
```

### CODE REVIEW

**Review by lhy1024 — COMMENTED** [review]
There are typos in the title, which should be "simulator: remove balance-region-scheduler until all stores put the regions to pd"

File: tools/pd-simulator/simulator/drive.go [hunk 1]
```diff
@@ -106,6 +110,12 @@ func (d *Driver) Prepare() error {
 func (d *Driver) Tick() {
 	d.tickCount++
 	d.raftEngine.stepRegions()
+	if d.tickCount <= 1 {
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/drive.go:113]: should be less than `storeHeartBeatPeriod`?

File: tools/pd-simulator/simulator/drive.go [hunk 2]
```diff
@@ -106,6 +110,12 @@ func (d *Driver) Prepare() error {
 func (d *Driver) Tick() {
 	d.tickCount++
 	d.raftEngine.stepRegions()
+	// simulator don't need any schedulers util all stores send their heartbeat.
+	if d.tickCount <= 1 {
+		d.client.RemoveScheduler(balanceRegionScheduler)
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/drive.go:115]: also need to remove balance leader scheduler?

**Review by lhy1024 — COMMENTED** [review]
rest LGTM

**Comment by nolouch** [comment]
cc @bufferflies

**Comment by lhy1024** [comment]
any update?

**Pull request closed without merging**